### PR TITLE
(9244) Add terraform support for vpn user accelerator

### PR DESCRIPTION
**Pull request by iggynopolis**

### CODE REVIEW

File: aviatrix/resource_vpn_user_accelerator.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
+package aviatrix
```
Comment (arvindsree) [inline aviatrix/resource_vpn_user_accelerator.go]: follow the naming convention as in 4.6 branch

**Comment by arvindsree** [comment]
update the erb file and add an acceptance test as well

**Comment by iggynopolis** [comment]
Will this be in the next release or the release after that?

**Comment by arvindsree** [comment]
the one after

File: aviatrix/resource_aviatrix_vpn_user_accelerator.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+		}
+		return fmt.Errorf("unable to read endpoint list for User Accelerator due to %v", err)
+	}
+	log.Printf("[DEBUG] elbList: %s", elbList)
+	if elbList != nil {
+		if goaviatrix.Contains(elbList, elbName) {
+			d.Set("elb_name", elbName)
+		} else {
+			d.SetId("")
+			return fmt.Errorf("unable to find elb in Vpn User Accelerator")
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_vpn_user_accelerator.go]: return nil and not error

File: aviatrix/resource_aviatrix_vpn_user_accelerator.go [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+		}
+	}
+
+	//d.SetId(strings.Replace(client.ControllerIP, ".", "-", -1))
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_vpn_user_accelerator.go]: remove commented code

File: aviatrix/resource_aviatrix_vpn_user_accelerator_test.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
+package aviatrix
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	//"github.com/hashicorp/terraform/helper/acctest"
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_vpn_user_accelerator_test.go]: remove commented code and run make imports

File: aviatrix/resource_aviatrix_vpn_user_accelerator_test.go [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+			},
+		},
+	})
+
+}
+
+func testAccVPNUserAcceleratorConfigBasic(elb string) string {
+	return fmt.Sprintf(`
+resource "aviatrix_vpn_user_accelerator" "test" {
+	elb_name = "%s"
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_vpn_user_accelerator_test.go]: Should minimize user parameters. You should just create a gateway with ELB and use that ELB for the accelerator

File: aviatrix/resource_aviatrix_vpn_user_accelerator.go [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+	log.Printf("[DEBUG] elbList: %s", elbList)
+	if elbList != nil {
+		if goaviatrix.Contains(elbList, elbName) {
+			d.Set("elb_name", elbName)
+		} else {
+			d.SetId("")
+			return fmt.Errorf("unable to find elb in Vpn User Accelerator")
+		}
+	}
+
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_vpn_user_accelerator.go]: return error/set nil here

**Pull request merged**